package.json

```diff
@@ -38,6 +38,7 @@
     "ember-load-initializers": "^0.5.1",
     "ember-resolver": "^2.0.3",
     "ember-suave": "^4.0.0",
+    "flexi-sustain": "1.2.0",
```

Comment:
What are the things that keep us from testing the sustain in it's own repo and just pulling in the other parts of flexi needed to do so as a dev-dependency there?